src/configure

```diff
@@ -1307,7 +1307,7 @@ if $enable_tcmalloc ; then
     TCMALLOC_ROOT=$(rel2abs ../tools/gperftools)
     echo "Configuring tcmalloc since --enable-tcmalloc"
     if [ "$(uname)" == "Linux" ]; then
-      echo "LDLIBS += -Wl,-rpath=$TCMALLOC_ROOT/lib -ltcmalloc_minimal" >> kaldi.mk
+      echo "LDLIBS += -Wl,-rpath=$TCMALLOC_ROOT/lib -L$TCMALLOC_ROOT/lib -ltcmalloc_minimal" >> kaldi.mk
```

Comment:
The CI should fail without `-L$TCMALLOC_ROOT/lib`.

Reply:
tcmalloc It's not enabled in CI. But your fix is just too correct not to merge it right away, thanks. :)